docs/chatterino.d.ts

```diff
@@ -375,4 +375,13 @@ declare namespace c2 {
         Original,
         Repost,
     }
+
+    function json_parse(
```

Comment:
@Mm2PL I'd really prefer if we focus on pure Lua support rather than trying to support transpiling.
How strongly do you want to support TS transpiling?